go/vt/mysqlctl/backup.go

```diff
@@ -58,6 +58,8 @@ const (
 const (
 	// replicationStartDeadline is the deadline for starting replication
 	replicationStartDeadline = 30
+
+	Error1193 = "Unknown system variable"
```

Comment:
There is a constant defined for this error code (not string). Can you use that instead, or should we worry about false positive hits?
It's defined here https://github.com/vitessio/vitess/blob/main/go/mysql/constants.go#L495

Reply:
I've migrated it to SQLError (as you have below) to align.